product_docs/docs/pgd/5.6/commit-scopes/synchronous_commit.mdx

```diff
@@ -1,9 +1,10 @@
 ---
-title: PGD Synchronous Commit
+title: Synchronous Commit
```

Comment:
The PGD was there to distinguish between legacy synchronous commit and PGD commit scope synchronous commit.